docs/other-sdks.md

```diff
@@ -0,0 +1,17 @@
+---
+id: other-sdks
+title: Other SDKs
+sidebar_label: Other SDKs
+---
+
+There is demand for other Temporal SDKs other than Go and Java. The user community has taken initiative to build some of these, but please note that they are not officially supported or endorsed by Temporal unless explicitly mentioned:
```

Comment:
- Paragraph could be more direct, consider:

Looking for SDKs in other languages? There are plans to support many other SDKs, but currently we only support Go and Java. This is a list of ones that are in the works but are not yet officially endorsed:

- In the context of the list, I am not sure what it means to be explicitly mentioned? I imagine that officially supported SDks would have a spot on the root of our docs pages, so anything still on this list wouldn't be officially endorsed?

Reply:
not officially endorsed yes, but the vision for why we explicitly mention is to point people toward "good enough" third party SDKs. obv a lot of placeholders now as we work it out.

Reply:
```suggestion
There is demand for other Temporal SDKs other than Go and Java. The user community has taken initiative to build some of these, but please note that they are not officially supported or endorsed by Temporal unless explicitly mentioned:
```